### ISSUE TITLE
spades-core running only with one processor

### ISSUE THREAD

**Issue by CristianMolinaQuiroz** [ablab/spades#1155]
### Description of bug

Hi, im running an hybrid assembly with spades and everything goes well without failures until now, but in the spades-core script in the assembly step of K77 its only using one processor even when it has been using the 60 processors that was setted at start in the other steps. It is taking so many time in this step and the spades.log show that its not even close to end the step that is running.

### spades.log

[spades.log](https://github.com/ablab/spades/files/12015348/spades.log)

### params.txt

[params.txt](https://github.com/ablab/spades/files/12015357/params.txt)

### SPAdes version

SPAdes version: 3.13.1

### Operating System

Ubuntu 20.04

### Python Version

Python version: 3.8.10

### Method of SPAdes installation

binaries

### No errors reported in spades.log

- [X] Yes

**Comment by CristianMolinaQuiroz** [comment]
Hi Anton, thanks for your quick answer. I didn't noticed that outputs lines until now and indeed seems to be strange. The insert size shown in the log could be affected due to the low amount of long reads that we have until now, but that we are increasing these weeks. We were trying to perform a pre assembly to set softwares and parameters and to test our computationals capabilities. But this low long reads amounts could lead to use only one processor in the assembly step that we are stucked?

The short reads library was prepared aiming to 390bp insert size and the estimated genome size is about 700MB.

Thanks in advance.
Regards

**Comment by asl** [comment]
> The insert size shown in the log could be affected due to the low amount of long reads that we have until now, but that we are increasing these weeks. 

It's all about your paired-end data. This look very strange (among other more subtle things) and it looks like it's paired-end data that is somehow wrong. Have you done QC before? Any read trimming / filtering, etc.?

**Comment by CristianMolinaQuiroz** [comment]
Yes, the reads were filtered with  Q30 quality filtered and the adapters were removed too.

**Comment by asl** [comment]
> Yes, the reads were filtered with Q30 quality filter and the adapters were removed too.

Have you used paired-end-aware trimmer tool? Do your left reads correspond to the right ones?
